client/cmd/list.go

```diff
@@ -0,0 +1,291 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+	"sort"
+	"strings"
+	"text/tabwriter"
+
+	"github.com/hoophq/hoop/client/cmd/styles"
+	clientconfig "github.com/hoophq/hoop/client/config"
+	"github.com/hoophq/hoop/common/httpclient"
+	"github.com/hoophq/hoop/common/log"
+	"github.com/hoophq/hoop/common/version"
+	"github.com/spf13/cobra"
+)
+
+var (
+	verboseFlag bool
+	outputFlag  string
+)
+
+var listExampleDesc = `hoop list
+hoop ls
+hoop list --verbose
+hoop list -o json`
+
+var listCmd = &cobra.Command{
+	Use:     "list",
+	Aliases: []string{"ls"},
+	Short:   "List available connections",
+	Long:    "Display connections that you have access to connect",
+	Example: listExampleDesc,
+	Run: func(cmd *cobra.Command, args []string) {
+		runList()
+	},
+}
+
+func init() {
+	listCmd.Flags().BoolVarP(&verboseFlag, "verbose", "v", false, "Show additional information including command")
+	listCmd.Flags().StringVarP(&outputFlag, "output", "o", "", "Output format. One of: (json)")
```

Comment:
As we talked about in the daily, let's introduce the enum type `-o wide` to display more fields instead of using the flag `--verbose`